admin/src/test/java/com/exadel/frs/EmbeddedPostgreSQLTest.java

```diff
@@ -0,0 +1,7 @@
+package com.exadel.frs;
+
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+
+@AutoConfigureEmbeddedDatabase(beanName = "dataSource")
+public class EmbeddedPostgreSQLTest {
```

Comment:
I suggest, to name class in more generally style like ```EmbeddedDatabaseTest``` and make that class abstract